tests/rules/cognitive-complexity.test.ts

```diff
@@ -491,9 +491,263 @@ ruleTester.run("cognitive-complexity 15", rule, {
   ],
 });
 
+ruleTester.run("overall-cognitive-complexity", rule, {
+  valid: [],
+  invalid: [
```

Comment:
testing all statements is not interesting, I would drop these tests. More interesting to test classes with methods, nested functions